src/components/AccountMenuDropdown.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react'
+import Menu from './Menu'
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group'
+
+class AccountMenuDropdown extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      menuActive: false
+    };
+  }
+
+  toggleMenu() {
+    let menuState = !this.state.menuActive;
+    this.setState({
+      menuActive: menuState
+    });
+  }
+
+  render() {
+    let menu
```

Comment:
We have this 75% fixed but if you could spare a quick moment to help us understand how to pass in the props from menuActive properly we would appreciate it.